main.py

```diff
@@ -126,36 +126,24 @@ def setup_target_task_training(args, target_tasks, model, strict):
         log.info("Loading existing model from %s...", best_path)
         load_model_state(model, best_path, args.cuda, task_names_to_avoid_loading, strict=strict)
     else:
-        # Look for target train checkpoints (available only if we're restoring from a run that
-        # already finished), then look for training checkpoints.
-
-        best_path = get_best_checkpoint_path(args.run_dir, "target_train")
-        if best_path:
-            # Continue target task training.
```

Comment:
Deleting intentionally? I know this may not have been fully supported, just making sure.

Reply:
Yes, this was not fully supported, and has to be changed now that we're saving our target tasks in multiple places depending on task.